internal/operands/template-validator/resources.go

```diff
@@ -0,0 +1,203 @@
+package template_validator
+
+import (
+	"fmt"
+
+	admission "k8s.io/api/admissionregistration/v1"
+	apps "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// Resources from kubevirt-template-validator version v0.6.6
+// TODO - move this code to the kubevirt-template-validator
+//        repository, and import it as a go module
+
+const (
+	containerPort         = 8443
+	kubevirtIo            = "kubevirt.io"
+	secretName            = "virt-template-validator-certs"
+	serviceAccountName    = "template-validator"
+	virtTemplateValidator = "virt-template-validator"
+)
+
+var commonLabels = map[string]string{
+	kubevirtIo: virtTemplateValidator,
+}
+
+func clusterRoleName(namespace string) string {
+	const clusterRoleName = "template:view"
+	return clusterRoleName + "-" + namespace
```

Comment:
Why do we integrate the namespace name into the clusterRole name?

Reply:
Multiple `SSP` resources can be created in different namespaces. In that case the operator creates multiple cluster roles, and they need different names. If we only used a single cluster role for all `SSPs`, then it would be tricky to implement deletion.